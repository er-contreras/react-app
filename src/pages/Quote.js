import React from 'react';

const Quote = () => { // eslint-disable-line
  return <div>hello from quote page</div>;
};

export default Quote;
