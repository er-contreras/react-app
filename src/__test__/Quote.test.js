import React from 'react';
import TestRenderer from 'react-test-renderer';
import { render, screen } from '@testing-library/react';
import Quote from '../pages/Quote';

describe('Quote test', () => {
  it('Should render same text passed into ...', () => {
    const tree = TestRenderer.create(<Quote />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('renders the main Text', () => {
    render(<Quote />);
    const heading = screen.getByText(/William Paul Thurston/i);
    expect(heading).toMatchSnapshot();
  });
});
