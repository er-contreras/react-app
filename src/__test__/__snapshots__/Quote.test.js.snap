// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quote test Should render same text passed into ... 1`] = `
<div
  id="quote"
>
  <p
    className="p-quote"
  >
    Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding. -William Paul Thurston
  </p>
</div>
`;

exports[`Quote test renders the main Text 1`] = `
<p
  class="p-quote"
>
  Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding. -William Paul Thurston
</p>
`;
